README.md

```diff
@@ -647,14 +647,382 @@ type VirtualMachine struct {
 
 To begin coding the basic execution functionality of our VM, we only need these basic fields, we will be adding more fields as we dive deeper into this guide.
 
-#### Instruction Decoding and Execution
+### Instruction Decoding and Execution
```

Comment:
Dont change this, this puts the `Instruction Decoding and Execution` on the same level as the `Code walkthrough/Write your own Cairo VM` title